src/view/hoc/PageComposer/FabPage/index.tsx

```diff
@@ -68,6 +68,7 @@ export default function createFabPage<InjectedProps>(WrappedComponent: React.Com
         <Fab
           onPress={this.emit}
           position="bottomRight"
+          style={{ backgroundColor: '#0277bd' }}
```

Comment:
so that it is linked to the theme use :

backgroundColor: MATERIAL_COLORS.secondary.normal

Reply:
I made these changes and pushed to master since we need this for tonights apk

Reply:
Sorry, I was unable to look at this until now. Thank you for making the appropriate changes. For some reason when I was trying to import MATERIAL_COLORS last night it wasn't working or building, so I resorted to using a hard-coded value that was equivalent to the theme. Thank you again.